madgraph/interface/reweight_interface.py

```diff
@@ -1756,72 +2254,72 @@ def create_standalone_directory(self, second=False):
             
             #multiparticles
             for name, content in self.banner.get('proc_card', 'multiparticles'):
-                try:
-                    mgcmd.exec_cmd("define %s = %s" % (name, content))
-                except madgraph.InvalidCmd:
-                    pass
-                    
-        if  second and 'tree_path' in self.dedicated_path:
-            files.ln(self.dedicated_path['tree_path'], path_me,name=data['paths'][0])
-            if 'virtual_path' in self.dedicated_path:
-                has_nlo=True
+                    try:
+                        mgcmd.exec_cmd("define %s = %s" % (name, content))
+                    except madgraph.InvalidCmd:
+                        pass
```

Comment:
```suggestion
                try:
                    mgcmd.exec_cmd("define %s = %s" % (name, content))
                except madgraph.InvalidCmd:
                    pass
```

no need of double tab here...